apps/maestro/src/features/SendInterchainToken/hooks/useInterchainTokenServiceTransferMutation.ts

```diff
@@ -72,7 +72,7 @@ export function useInterchainTokenServiceTransferMutation(
     reset: resetInterchainTransferMutation,
   } = useInterchainTokenServiceInterchainTransfer({
     address: NEXT_PUBLIC_INTERCHAIN_TOKEN_SERVICE_ADDRESS,
-    value: BigInt(config.gas ?? 0) * BigInt(2),
```

Comment:
this 2 is meant to be a multiplier so we pay enough cross chain gas, right? `config.gas` itself should be multiplied by 2